Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -53,18 +53,17 @@ namespace Hazel {
 		glBindVertexArray(m_RendererID);
 		vertexBuffer->Bind();
 
-		uint32_t index = 0;
 		const auto& layout = vertexBuffer->GetLayout();
 		for (const auto& element : layout)
 		{
-			glEnableVertexAttribArray(index);
-			glVertexAttribPointer(index,
+			glEnableVertexAttribArray(m_VertexBufferIndex);
+			glVertexAttribPointer(m_VertexBufferIndex,
 				element.GetComponentCount(),
 				ShaderDataTypeToOpenGLBaseType(element.Type),
 				element.Normalized ? GL_TRUE : GL_FALSE,
 				layout.GetStride(),
 				(const void*)(intptr_t)element.Offset);
-			index++;
+			m_VertexBufferIndex++;
```

Comment:
This is not needed to get Hazel up and running on Linux.

Reply:
This is not done by this PR. This was done in TheCherno/Hazel#102 and TheCherno/Hazel#105. We just upstreamed Hazel to be up to date with the latest changes.